package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "pnpm generate && next dev",
-    "build": "pnpm generate && next build",
+    "dev": "pnpm generate && NODE_OPTIONS='--inspect' next dev",
+    "build": "next build",
```

Comment:
@jwm0 `pnpm generate` must in in `build` as it generates the GraphQL stub: check the failed build